website_compassion/models/invoice_line.py

```diff
@@ -6,9 +6,15 @@
 #    The licence is in the file __manifest__.py
 #
 ##############################################################################
-from odoo import models
+from odoo import models, fields
 
 
 class AccountInvoiceLine(models.Model):
     _name = "account.invoice.line"
     _inherit = ["account.invoice.line", "translatable.model"]
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.invoice"
+    # If true, cancel the invoice if the linked payment transaction is cancelled
```

Comment:
This string can be put in the `help` attribute of the field
```suggestion
    # If true, cancel the invoice if the linked payment transaction is cancelled
```